crmsh/sbd.py

```diff
@@ -241,6 +250,30 @@ def _enable_sbd_service(self):
             # in init process
             bootstrap.invoke("systemctl enable sbd.service")
 
+    def _adjust_systemd(self):
+        """
+        Adjust start timeout for sbd when set SBD_DELAY_START
+        """
+        if not self._delay_start:
+            return
+
+        # TimeoutStartUSec default is 1min 30s, need to parse as seconds
+        cmd = "systemctl show service -p TimeoutStartUSec"
```

Comment:
It should be more sensible to just check this for `sbd.service`.

Reply:
`systemctl show sbd.service -p TimeoutStartUSec`?

Reply:
> It should be more sensible to directly check this for `sbd.service`. And parsing could be easier by supplying `--value` option here.

aha, yes, make sense, `systemctl show -p TimeoutStartUSec sbd --value`

We have hard time to find if systemd can generate "90s" instead of "1m 30sec", but systemd don't provide, sigh